WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -47,6 +47,20 @@ def check_results
     @results_validator = CompetitionResultsValidator.new(@competition.id, true)
   end
 
+  def clear_results_submission
+    # Just clear the "results_submitted_at" field to let the Delegate submit
+    # the results again. We don't actually want to clear InboxResult and InboxPerson.
+    @competition = competition_from_params
+
+    if @competition.results_submitted? && !@competition.results_posted?
+      @competition.update_attributes(results_submitted_at: nil)
+      flash[:success] = "Results submission cleared."
+    else
+      flash[:danger] = "Could not clear the results submission. Maybe results are alredy posted, or there is no submission."
```

Comment:
Perhaps we could have two different messages depending on the conditions? Rendering `Maybe` is kinda funny given we know what's the reason ^^

Reply:
It's intentional: my thought process was that reaching this error is not possible with a "regular" usage of the website, therefore the person reaching that error deserved a "Maybe" error :p

Reply:
Haha, fair enough!

Reply:
nit: alredy -> already